internal/server/middleware/grpc/middleware.go

```diff
@@ -314,8 +319,15 @@ func AuditUnaryInterceptor(logger *zap.Logger) grpc.UnaryServerInterceptor {
 
 		defer func() {
 			if event != nil {
-				span := trace.SpanFromContext(ctx)
-				span.AddEvent("event", trace.WithAttributes(event.DecodeToAttributes()...))
+				ts := string(event.Type)
+				as := string(event.Action)
+				eventPair := fmt.Sprintf("%s:%s", ts, as[:len(as)-1])
```

Comment:
I wonder if perhaps we should just make folks use the past tense verb form here and keep it consistent.
Just imagining e.g. if we added something like `order` for reordering, the past tense would be `ordered` or `reordered` and then stripping the last character wouldn't make sense. Plus this is a potential bounds problem if the action was accidentally empty.

Reply:
@GeorgeMac Yeah good point, I thought about this as well. I can use the past tense verb 👍